### ISSUE TITLE
new update error

### ISSUE THREAD

**Issue by yceballost** [Martvvliet/homebridge-mi-flora-filtered#7]
```
Error: Cannot add a Characteristic with the same UUID as another Characteristic in this Service: undefined
    at PlantSensor.Service._this.addCharacteristic (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Service.ts:190:15)
    at PlantSensor.Service.getCharacteristic (/usr/lib/node_modules/homebridge/node_modules/hap-nodejs/src/lib/Service.ts:292:23)
    at MiFlowerCarePlugin.setUpServices (/usr/lib/node_modules/homebridge-mi-flora-filtered/index.js:337:29)
    at new MiFlowerCarePlugin (/usr/lib/node_modules/homebridge-mi-flora-filtered/index.js:54:10)
    at /usr/lib/node_modules/homebridge/src/server.ts:356:50
    at Array.forEach (<anonymous>)
    at Server._loadAccessories (/usr/lib/node_modules/homebridge/src/server.ts:329:29)
    at Server.start (/usr/lib/node_modules/homebridge/src/server.ts:155:12)
    at cli (/usr/lib/node_modules/homebridge/src/cli.ts:80:10)
    at Object.<anonymous> (/usr/lib/node_modules/homebridge/bin/homebridge:17:22)
```

**Comment by Martvvliet** [comment]
Yup, had the same issue. Sorry, should be fixed in 1.3.4

**Comment by yceballost** [comment]
Solved!
